content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -51,8 +51,13 @@ If it writes files or dirs, they can be defined as <abbr>outputs</abbr>
 ### Parameter dependencies
 
 [Parameters](/doc/command-reference/params) are a special type of stage
```

Comment:
I'm not sure we need any of the changes to this user guide page. It otherwise looks like a very brief overview of parameters, and this seems like too much detail.

Reply:
Oh I see you did update the schema ref. already @skshetry 👍🏼 

@dberenbaum I do think we need to update it since the schema changed. The detail given now may be too much but we can address that...

Reply:
> the schema changed 

p.s. does https://github.com/iterative/dvcyaml-schema need an update?

Reply:
> > the schema changed
> 
> p.s. does https://github.com/iterative/dvcyaml-schema need an update?

Done in https://github.com/iterative/dvcyaml-schema/pull/12